NaReguaApp/src/routes/index.js

```diff
@@ -0,0 +1,24 @@
+import React, { useContext } from 'react';
```

Comment:
Codacy has a fix for the issue: [Replace `·useContext·` with `useContext`](https://app.codacy.com/gh/UFAPE-Projetao20202/naregua_front/pullRequest?prid=8386487)
```suggestion
import React, {useContext} from 'react';
```